gradio/pipelines_utils.py

```diff
@@ -26,21 +26,14 @@ def is_transformers_pipeline_type(pipeline, class_name: str):
     # version of the transformers library that the user has installed.
     if is_transformers_pipeline_type(pipeline, "AudioClassificationPipeline"):
         return {
-            "inputs": components.Audio(
-                sources=["microphone"],
-                type="filepath",
-                label="Input",
-                render=False,
-            ),
+            "inputs": components.Audio(type="filepath", label="Input", render=False),
             "outputs": components.Label(label="Class", render=False),
             "preprocess": lambda i: {"inputs": i},
-            "postprocess": lambda r: {i["label"].split(", ")[0]: i["score"] for i in r},
```

Comment:
This transforms a label name like "tiger, Panthera tigris" into just "tiger", which looks not good.

On `main`:
![CleanShot 2024-04-19 at 16 27 16@2x](https://github.com/gradio-app/gradio/assets/3135397/df06100e-532c-4d27-8303-238bad5d838f)

This change:
![CleanShot 2024-04-19 at 16 27 50@2x](https://github.com/gradio-app/gradio/assets/3135397/71b7b787-8609-403f-8063-bfb2cf7cdae9)